### ISSUE TITLE
Banner doesn't resize on mobile

### ISSUE THREAD

**Issue by ragu-manjegowda** [arkadianriver/arkadianriver.com#28]
Hi, 
I noticed that banner.jpg covers full screen on landing page when viewed in desktop. 
However it doesn't resize properly on mobile. 
Can you help me understand how to configure it so that landing page is consistent between desktop and mobile?  
Thanks

**Comment by arkadianriver** [comment]
Yeah, the aspect ratio of the image on mobile is more square than 16:9, so the side edges of the image will spill out of view on mobile.

Choosing images is a bit tricky. I typically will resize my browser window throughout all the sizes, and watch the breakpoints at various sizes to be sure the image looks good no matter how large or small the browser is. And, I use the mobile preview in the Chrome/Firefox page inspector. If the image doesn't look good in all aspects, I try cropping it so that it does.

The most difficult to resize are pictures with people, such as the topic in my blog about my dad. It's still not perfect in all aspects, sometime cutting off the top of his head, but it's decent.

I'll look closer at the theme and see if things can be changed programatically, but most likely the best I'll be able to do without significantly changing the spectral theme is write up a guide on how best to size your images so that they look good in all aspects.
